packages/sns/lib/utils/stsUtils.spec.ts

```diff
@@ -0,0 +1,22 @@
+import type { STSClient } from '@aws-sdk/client-sts'
+import { beforeAll, describe, expect, it } from 'vitest'
+import { registerDependencies } from '../../test/utils/testContext'
+import { buildTopicArn } from './stsUtils'
+
+describe('stsUtils', () => {
+  let stsClient: STSClient
+
+  beforeAll(async () => {
+    const diContainer = await registerDependencies({}, false)
+    stsClient = diContainer.cradle.stsClient
+  })
+
+  describe('buildTopicArn', () => {
+    it('build ARN for topic', async () => {
+      const topicName = 'my-test-topic'
+      await expect(buildTopicArn(stsClient, topicName)).resolves.toMatchInlineSnapshot(
```

Comment:
might be good to also create a topic with the same name within the test, and compare actual arn to manually built one. then we will break if AWS will change the structure at some point

Reply:
Good idea! will add it 🙌 thanks igor 🙇